app/assets/javascripts/admin_procedures_modal.js

```diff
@@ -52,7 +52,7 @@ function path_validation(el) {
 }
 
 function validatePath(path) {
-  var re = /^[a-z0-9_]{3,30}$/;
+  var re = /^[a-z0-9_\-]{3,50}$/;
```

Comment:
ajoute "and can be longer" à ton commit message pour réfleter aussi le changement de longueur ? (ou extrait-le dans un 2e commit)